### ISSUE TITLE
Design: redesign the list to download resources (files)

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#829]
Following the merging of this **PR #828** to remove links to Drive, leaving only downloadable files and no longer a Drive preview, we need to redesign the list where the user can download resources (kits, templates, ...). We need a preview and a more usable way to recognize what and how to download. 

**Opportunities:** 
- to evaluate every possible automations (file size, file icon, thumbnail preview, file last update (if near), ...) cc @bfabio 

**Known issues:**
- [x] We currently does not have a PPTX icon in Bootstrap Italy. Quick solution adopted: we used the generic `it-file-slides` icon, but it is currently confusing. Added.
- [ ] Hover underline on title + text

![example](https://github.com/italia/designers.italia.it/assets/6736103/e20f7e35-58b3-4b29-9e18-7ebec7a8f1d1)
[Source](https://designers-italia-lz8y0f0py-dip-trasformazione-digitale.vercel.app/risorse-per-progettare/comprendere/analisi-del-contesto/schematizza-il-contesto-d-uso/)

**Comment by Lorenzo-Pappalardo** [comment]
Hi, can you provide more details about how you would like to redesign this? Possibly attaching some mock-ups for reference.
Thanks 😄
